_component_design/pills.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Pills
+parent: Content Areas
+secondary: Editing
+layout: component
+category: Components
+usage: >
+  Pills are small, pill-shaped containers that indicate selection and can be removed. They are useful for their small size, and for the ease with which they can be removed. As selection indicators (not selectors themselves), they have a broad set of possible applications. 
```

Comment:
Remove... "and can be removed." Redundant considering the next sentence says, "for the was with which they can be removed."

Reply:
Remove comma between... "for their small size and for the ease."

Reply:
updated.